scripts/StartDB1000N.bat

```diff
@@ -0,0 +1,28 @@
+echo $SaveFolder = $args[0]                                                       >  %temp%\GetDB1000N.ps1
```

Comment:
@Devaniti is there any option to add a short comment in this script, what does it for? For people who are not familiar with Windows?

Reply:
makes sense

Reply:
Let's do it as a separate PR